packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -336,15 +366,21 @@ export interface ApplicationWithRepositories extends Application {
     repo: Class<R>,
     name?: string,
   ): Binding<R>;
+
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   getRepository<R extends Repository<any>>(repo: Class<R>): Promise<R>;
+
   dataSource<D extends juggler.DataSource>(
     dataSource: Class<D> | D,
     name?: string,
   ): Binding<D>;
+
   model<M extends Class<unknown>>(modelClass: M): Binding<M>;
+
   component(component: Class<unknown>, name?: string): Binding;
+
   mountComponentRepositories(component: Class<unknown>): void;
+
   migrateSchema(options?: SchemaMigrationOptions): Promise<void>;
```

Comment:
Please revert the addition of new lines above, it seems to me as unrelated to fixing datasource migration.